geth-dev-node/UTC--2017-10-16T19-13-42.464934000Z--c5ed899b0878656feb06467e2e9ede3ae73cbcb7

```diff
@@ -0,0 +1 @@
+{"address":"c5ed899b0878656feb06467e2e9ede3ae73cbcb7","crypto":{"cipher":"aes-128-ctr","ciphertext":"4c8da4df20e90fa044d1679c8e29dffb4d5cdf126770dccddfb3136da5ce74ee","cipherparams":{"iv":"0fcfef299dd9faa41163ff32180e17f5"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"951f29c628ef1f71c25732b43c14fae581928d68bc9f944b6878e10c7656cf32"},"mac":"854cfdebf1941d5c0ec9ebc5296c796a3254029f13ab57a4218ee200e47be9c7"},"id":"97c0f84f-bc0f-4e06-a466-67716ca4a9aa","version":3}
```

Comment:
In VSCode you can set the filetype to JSON and then open the command pallet and hit "Format Document".  This will pretty print this key file as JSON which will make it easier to read and diff.